src/geometry/manifold/instances/sphere.lean

```diff
@@ -414,6 +454,66 @@ begin
   exact cont_mdiff_coe_sphere,
 end
 
+/-- Consider the differential of the sphere in `E` at the point `v` as a continuous linear map from
```

Comment:
`the differential of the sphere` sounds weird to me. `the differential of the inclusion of the sphere`? Same thing below.

Reply:
Thanks for catching this!